cloudvolume/cloudvolume.py

```diff
@@ -105,7 +105,7 @@ class CloudVolume(object):
     non_aligned_writes: (bool) Enable non-aligned writes. Not multiprocessing safe without careful design.
       When not enabled, a ValueError is thrown for non-aligned writes.
   """
-  def __init__(self, cloudpath, mip=0, bounded=True, autocrop=False, fill_missing=False, 
+  def __init__(self, cloudpath, mip=0, resolution=None, bounded=True, autocrop=False, fill_missing=False, 
```

Comment:
You should put the new `kwarg` at the very end, to ensure backward compatibility. Otherwise existing code, where the _keyword_ arguments are passed as _positional_ arguments, will start to behave strangely.

Reply:
Maybe even better, just handle both code paths with the single `mip` parameter? They describe the same thing, so it wouldn't make sense to use both at the same time; and their input types don't overlap.

Reply:
Responded to these in the main thread - will do. Thanks for the feedback